test/pwave.jl

```diff
@@ -10,7 +10,13 @@ unitcell = (2, 2)
 H = pwave_superconductor(InfiniteSquare(unitcell...))
 χbond = 2
 χenv = 16
-ctm_alg = CTMRG(; tol=1e-8, maxiter=150, verbosity=2, ctmrgscheme=:sequential)
+ctm_alg = CTMRG(;
+    tol=1e-8,
+    maxiter=150,
+    verbosity=2,
+    ctmrgscheme=:simultaneous,
+    svd_alg=SVDAdjoint(; rrule_alg=Arnoldi(; tol=1e-9, krylovdim=χenv + 30)),
+)
```

Comment:
at the risk of sounding like a broken record, I really don't like that we have to change the settings of all algorithms on every single update of the package. We really have to get some better defaults in, because this is not really feasible in the long run. (not something that has to be fixed in this pr)

Reply:
You definitely have a point in repeating that and I completely agree :-) Actually, for most of the tests we should already be able to replace all settings with the default ones I just haven't gotten around to change them. I just thought that I would wait until we anyway redesign our interface and code up some kind of default parameter selection thing.